### PR TITLE
Add npc-collection-log plugin

### PR DESCRIPTION
**Pull request by ElOsoGroso**
NPC Collection Log

This plugin shows a panel when you initiate combat with any NPC, showing information about which items are possible to receive from them, and which ones you've already got in your collection log, or in your bank.

### CODE REVIEW

**Comment by raiyni** [comment]
Would be helpful if you scaled down the icon size.

**Comment by ElOsoGroso** [comment]
> Would be helpful if you scaled down the icon size.

The nav-icon.png in resources or icon.png? I assume the nav-icon one. Yeah I can do that.

**Comment by ElOsoGroso** [comment]
> Would be helpful if you scaled down the icon size.

That's been resized.

File: plugins/npc-collection-log [hunk 1]
```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/ElOsoGroso/NPCCollectionLog.git
+commit=988dce503948f84f6d9f8826309ead808c3a3294
+
```
Comment (Nightfirecat) [inline plugins/npc-collection-log:3]:
```suggestion
warning=This plugin submits your account hash and IP address to a server not controlled or verified by the RuneLite developers.
```

**Review by Nightfirecat — CHANGES_REQUESTED** [review]

**Comment by ElOsoGroso** [comment]
After this merge... it seems to be immediately crashing upon login or installing the plugin. Locally I don't see the same behavior at all @Nightfirecat

**Pull request merged**